packages/postcss-zindex/src/index.js

```diff
@@ -4,7 +4,7 @@ function pluginCreator(opts = {}) {
   return {
     postcssPlugin: 'postcss-zindex',
     prepare() {
-      const cache = new LayerCache(opts);
+      const cache = new LayerCache();
```

Comment:
opts is not used in the constructor, it's only a parameter for the `optimizeValues` method used below.